docs/developer/checkout/finalizing.mdx

```diff
@@ -151,7 +151,7 @@ The purpose of the operation is to ensure this checkout is correct. The validati
 
 1. The required addresses are valid.
 
-2. All selected products are in stock (while making the purchase, another user could already buy the last available item).
+2. All selected products are in stock (while making the purchase, another user could already buy the last available item), except variants that have disabled stock tracking which can have negative stock.
```

Comment:
Is the negative part important here? I think it would be easier to say, "except for variants that have stock tracking disabled in which case the stock level does not matter."

Reply:
I agree, but my intention was to add "negative stock" as a keyword to be discoverable via search as this is a common question

Reply:
Can we then say "zero or negative?"